SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -3031,7 +3038,7 @@ void USpatialNetDriver::RefreshActorDormancy(AActor* Actor, bool bMakeDormant)
 	const bool bHasAuthority = HasServerAuthority(EntityId);
 	if (bHasAuthority == false)
 	{
-		UE_LOG(LogSpatialOSNetDriver, Warning, TEXT("Unable to flush dormancy on actor (%s) without authority"), *Actor->GetName());
+		UE_LOG(LogSpatialOSNetDriver, Verbose, TEXT("Unable to flush dormancy on actor (%s) without authority"), *Actor->GetName());
```

Comment:
Why this change?

Reply:
Since the entity id is now allocated, it doesn't go into the previous if statement, so I changed this one from warning to verbose just like the previous one

Reply:
I am not sure what you mean by 'Since the entity id is now allocated'?

Reply:
@m-samiec you wrote this code, can you share why this originally a warning?

Reply:
Running this code without authority would imply that a non-auth server is attempting to modify the entity. Which suggests an error further up the chain.

The two checks are (1) do we have an entity id, and (2) do I have auth over that entity, I can see with this PR the gap between 1 and 2 has grown, so this warning is more likely to occur, however I'd like to know what the test scenario is that results in the warning as it would suggest we're modifying dormancy before we should.

If the call to modify dormancy is valid, the fix seems to be that we add an additional check between (1) and (2) which is "Do I have the entity in my view?". If not, we can return with a verbose error like we did for (1), and keep (2) as a warning.

Reply:
This would do it `Connection->GetCoordinator().GetView().HasEntity(EntityId)`